### ISSUE TITLE
Bokeh jQuery overriding previously loaded jQuery

### ISSUE THREAD

**Issue by mistakevin** [bokeh/bokeh#2436]
Presumably an issue with [$.noConflict()](http://api.jquery.com/jQuery.noConflict/)

**Comment by damianavila** [comment]
ping @mattpap to be aware of this issue...

**Comment by colinfang** [comment]
I noticed that earlier today as well!

```{html}
<html>
<head>
    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
  
    <script src="http://cdn.pydata.org/bokeh/release/bokeh-0.9.0.min.js"></script>

</head>
<body>
    <script>console.log($.fn); console.log(jQuery.fn)</script>
</body>
</html>
```

It says `bokeh` defines `$` to be jquery 2.1.3

**Comment by damianavila** [comment]
Solved by #2489
